security/azure/sql_server_va_emails/sql_server_va_emails.pt

```diff
@@ -165,30 +408,50 @@ end
 # Policy
 ###############################################################################
 
-policy "policy_sql_server_va_emails" do
-  validate $ds_bad_sql_servers do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure SQL Server(s) Found Without VA Notification Emails"
+policy "pol_bad_azure_sql_servers" do
+  validate_each $ds_bad_azure_sql_servers do
+    summary_template "{{ with index data 0 }}{{ .policy_name }}{{ end }}: {{ len data }} Azure SQL Servers Whose Vulnerability Assessment Lacks Email Notifications Found"
+    check logic_or($ds_parent_policy_terminated, eq(val(item, "id"), ""))
     escalate $esc_email
-    check eq(size(data),0)
+    hash_exclude "tags"
     export do
-      field "name" do
-        label "Name"
-      end
-      field "location" do
-        label "Location"
-      end
-      field "kind" do
-        label "Kind"
-      end
-      field "subscriptionId" do
+      resource_level true
+      field "accountID" do
         label "Subscription ID"
       end
-      field "subscriptionName" do
+      field "accountName" do
         label "Subscription Name"
       end
+      field "resourceGroup" do
+        label "Resource Group"
+      end
+      field "resourceName" do
+        label "Resource Name"
+      end
       field "id" do
         label "Resource ID"
       end
+      field "region" do
+        label "Region"
+      end
+      field "tags" do
+        label "Tags"
+      end
+      field "vaState" do
+        label "Vulnerability Assessment Status"
+      end
+      field "fqdn" do
+        label "FQDN"
+      end
+      field "state" do
+        label "State"
+      end
+      field "version" do
+        label "Version"
+      end
+      field "service" do
+        label "Service"
+      end
```

Comment:
Should "vaEmails" field also be here as part of the export fields?

Reply:
No, because it is always empty by definition if its in the results.